test/e2e_node/e2e_node_suite_test.go

```diff
@@ -61,6 +61,7 @@ var runServicesMode = flag.Bool("run-services-mode", false, "If true, only run s
 var runKubeletMode = flag.Bool("run-kubelet-mode", false, "If true, only start kubelet, and not run test.")
 var systemValidateMode = flag.Bool("system-validate-mode", false, "If true, only run system validation in current process, and not run test.")
 var systemSpecFile = flag.String("system-spec-file", "", "The name of the system spec file that will be used for node conformance test. If it's unspecified or empty, the default system spec (system.DefaultSysSpec) will be used.")
+var kubectlPath = flag.String("kubectl-path", "kubectl", "The kubectl binary to use. For development, you might use 'cluster/kubectl.sh' here.")
```

Comment:
Rather than adding the flag here, move the cluster flag to common flags:
https://github.com/kubernetes/kubernetes/blob/0c9288e97d1a7ac05aa8175204b86a161661520a/test/e2e/framework/test_context.go#L311